NextcloudTalk/RoomCreationTableViewController.swift

```diff
@@ -0,0 +1,768 @@
+                roomVisibilityOptionCell.imageView?.image = self.roomPassword.isEmpty ? UIImage(systemName: "lock.open") : UIImage(systemName: "lock")
+            case RoomVisibilityOption.kOpenConversationOption.rawValue:
+                roomVisibilityOptionCell = tableView.dequeueReusableCell(withIdentifier: "OpenConversationCellIdentifier") ?? UITableViewCell(style: .default, reuseIdentifier: "OpenConversationCellIdentifier")
+                roomVisibilityOptionCell.textLabel?.text = NSLocalizedString("Open conversation to registered users", comment: "")
+                let optionSwicth = UISwitch()
+                optionSwicth.isOn = self.isOpenConversation
+                optionSwicth.addTarget(self, action: #selector(openConversationValueChanged(_:)), for: .valueChanged)
+                roomVisibilityOptionCell.accessoryView = optionSwicth
+                roomVisibilityOptionCell.imageView?.image = UIImage(systemName: "list.bullet")
+            case RoomVisibilityOption.kOpenConversationGuestsOption.rawValue:
+                roomVisibilityOptionCell = tableView.dequeueReusableCell(withIdentifier: "OpenConversationGuestsCellIdentifier") ?? UITableViewCell(style: .default, reuseIdentifier: "OpenConversationGuestsCellIdentifier")
+                roomVisibilityOptionCell.textLabel?.text = NSLocalizedString("Also open to guest app users", comment: "")
+                let optionSwicth = UISwitch()
+                optionSwicth.isOn = self.isOpenForGuests
+                optionSwicth.addTarget(self, action: #selector(openForGuestsValueChanged(_:)), for: .valueChanged)
+                roomVisibilityOptionCell.accessoryView = optionSwicth
+                roomVisibilityOptionCell.imageView?.image = UIImage(systemName: "list.bullet")
+                roomVisibilityOptionCell.imageView?.isHidden = true
+            default:
+                break
+            }
+
+            roomVisibilityOptionCell.selectionStyle = .none
+            roomVisibilityOptionCell.imageView?.tintColor = .secondaryLabel
+            roomVisibilityOptionCell.imageView?.contentMode = .scaleAspectFit
+            roomVisibilityOptionCell.textLabel?.numberOfLines = 0
+
+            return roomVisibilityOptionCell
+        }
+
+        return UITableViewCell()
+    }
+
+    override func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        let sections = getRoomCreationSections()
+        let roomCreationSection = sections[section]
+
+        if roomCreationSection == RoomCreationSection.kRoomNameSection.rawValue {
+            return NSLocalizedString("Name", comment: "")
+        } else if roomCreationSection == RoomCreationSection.kRoomDescriptionSection.rawValue {
+            return NSLocalizedString("Description", comment: "")
+        } else if roomCreationSection == RoomCreationSection.kRoomVisibilitySection.rawValue {
+            return NSLocalizedString("Visibility", comment: "")
```

Comment:
Should we add a comment here, to make it more clear what this Visibility is about?